hub/management/commands/import_council_data.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss-code"
+    message = "Importing council population data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "data_type": "integer",
+        "category": "place",
+        "subcategory": "",
+        "release_date": "February 2023",
+        "label": "Council population",
+        "source_label": "Data from mySociety.",
```

Comment:
Is this the "estimated 2020 population" from https://pages.mysociety.org/uk_local_authority_names_and_codes/datasets/uk_la_future/latest ? Or something else? Would be nice to link to the _actual_ source of the data, rather than our homepage, generically.

Reply:
If this _is_ indeed the `pop-2020`, then the `"release_date": "February 2023"` above should change too.